core/templates/dev/head/css/oppia.css

```diff
@@ -2140,3 +2145,90 @@ md-card.preview-conversation-skin-supplemental-card {
 .trained-feedback-selection:hover {
   color: #00897B;
 }
+
+.oppia-modal-information-card .modal-dialog {
+ width: 400px;
+ margin:100px auto;
```

Comment:
general nits, here and below: (i) single space after colon, (ii) alphabetize attrs within each css rule, (iii) prefix css class names with 'oppia-' to distinguish from bootstrap/material-design/other stuff.